src/components/Popup/Popup.vue

```diff
@@ -52,17 +57,18 @@ export default defineComponent({
       required: false,
       default: 'center'
     },
-    title: {
-      type: String,
-      required: false
-    },
     height: {
       type: String,
       required: false
     },
     width: {
       type: String,
       required: false
+    },
+    showCloseButton: {
+      type: Boolean,
+      required: false,
+      default: false
```

Comment:
you correctly have your boolean prop defaulted to `false` as [expected](https://vuejs.org/v2/guide/components-props.html#Passing-a-Boolean), but I thought hiding by default seemed weird so I checked in with @joeclayallday. Joe tells me that our designs currently have no "x" to close with no plans to add in the future. I would be in favor of just removing the "x" completely